manager/allocator/allocator_suite_test.go

```diff
@@ -0,0 +1,14 @@
+package allocator
+
+import (
+	. "github.com/onsi/ginkgo"
```

Comment:
Just a question: what's the philosophy going forward? Should new integration tests use gingko?